ndk/src/native_window.rs

```diff
@@ -99,7 +99,8 @@ impl NativeWindow {
         height: i32,
         format: Option<HardwareBufferFormat>,
     ) -> Result<()> {
-        let format: u32 = format.map_or(0, |f| f.into());
+        let format: Option<ffi::AHardwareBuffer_Format> = format.map(|f| f.into());
+        let format: u32 = format.map_or(0, |f| f.0);
```

Comment:
Can we just write this as:

```suggestion
        let format = format.map_or(0, |f| ffi::AHardwareBuffer_Format::from(f).0);
```

Without additional explicit types etc?

Reply:
:+1:. Perhaps a similar thing (minus the `map_or()`) in `into_native()`?